content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -25,11 +25,20 @@ This provides a foundation to understand the steps that Data Flow will automate
 
 ## Development
 
+The following sections describe how to build this stream from scratch. If you prefer, you can download a zip file containing the sources for the sample stream, unzip it, and proceed to the [build](#building) step.
+
+You can can [download the project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/standalone-stream-rabbitmq/dist/usage-cost-stream-rabbit.zip?raw=true) from your browser, or from the command-line:
+
+```bash
+wget https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/standalone-stream-rabbitmq/dist/usage-cost-stream-rabbit.zip?raw=true -O usage-cost-stream-rabbit.zip
```

Comment:
don't think we need to have wget if we have the download link, which could be included in the previous sentence.